server/db/model/Application.js

```diff
@@ -1,4 +1,41 @@
-/* eslint-disable */
-var mongoose = require('../index.js'),
-    config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+var mongoose = require('../index.js'), User = require('./User.js');
+
+// Define the document Schema
+var schema = new mongoose.Schema({
+    user: String,
+    experience: {
+        type: String,
+        enum: ['veteran', 'novice']
+    }
```

Comment:
@SigmanZero should we include profile stuff here too? If we don't there is no enforcing that a user has his/her birthdate, university, major in their profile